### ISSUE TITLE
Fanz.ee_возможно повторно пройти опрос

### ISSUE THREAD

**Issue by TONFOND** [teamfanzee/community#35]
### Bug Type

UX

### Reproduction steps

1. выбираем опрос
2. проходим его с ошибками - стараемся ошибиться везде
3. сайт показывает правильные ответы
4. после завершения опроса сайт предлагает пройти опрос повторно
5. проходишь повторно и вносишь правильные ответы
6. получаешь награду

тоже самое касается когда ты не укладываешься во время, можно повторно пройти опрос, зная заранее вопросы

### Actual result

прошел опрос с нулевым результатом и близким к нулю, сайт тебе предложен пройти опрос еще раз ))

### Expected result

ошибся, значит ошибся, не позволять проходить повторно
иначе пользователи могут манипулировать уязвимостью

### Suggested Severity

High

### Device

Desktop (please complete the following information):
OS: iOS 12.6
Browser Google
Version 105.0.5195.102

Smartphone (please complete the following information):

Device: iPhone Xs
OS: iOS 15.2.1
Browser Safari

### Additional Context

https://user-images.githubusercontent.com/113825222/192141567-0bed5ec3-24e2-4b33-8714-5d2f62221c00.mov

**Comment by teamfanzee** [comment]
Да, это не баг а фича. Пока фанатов немного и они только вникают что к чему мы решили чуть упростить им задачу, чтобы не стопорить джорни. Это поменяется со временем, но в данный момент поведение соответствует нашим задачам

**Comment by TONFOND** [comment]
Хорошо, пусть будет так )
Объективно, тестировать продукт без скриптов и понимания, что должно быть на выходе - сложно
